keps/prod-readiness/sig-node/2043.yaml

```diff
@@ -0,0 +1,5 @@
+kep-number: 2043
+stable:
+  approver:
+    - "@wojtek-t"
+    - "@sig-node-leads"
```

Comment:
Only wojtek should be listed here. He is the only PRR approver. You still need approval from the SIG leads for the KEP overall.